p2p/src/main/kotlin/maru/p2p/MaruPeerManager.kt

```diff
@@ -44,14 +52,18 @@ class MaruPeerManager(
     }, 20000, 20000, TimeUnit.MILLISECONDS)
   }
 
+  val blocksBehindThreshold: ULong = 32uL
```

Comment:
Can this be made configurable. Doesn't need to be in the file config but being in the `P2P` data class so it can be changed in testing would be helpful.

Reply:
Done